site/docs/spark-queries.md

```diff
@@ -219,9 +221,33 @@ SELECT * FROM prod.db.table.files
 +-------------------------------------------------------------------------+-------------+--------------+--------------------+--------------------+------------------+-------------------+------------------+-----------------+-----------------+--------------+---------------+
 ```
 
+### All Data Files
```

Comment:
I think it makes sense to have a separate section for the "all" tables. These aren't needed nearly as often and require warnings, as you have in these sections.